mon-pix/app/components/routes/campaigns/invited/fill-in-participant-external-id.hbs

```diff
@@ -14,7 +14,7 @@
     <div class="fill-in-participant-external-id-form__section">
       <div class="fill-in-participant-external-id__form-field">
         <label for="id-pix-label">{{@campaign.idPixLabel}}</label>
-        <Input required @id="id-pix-label" @value={{this.participantExternalId}} />
+        <Input required id="id-pix-label" @value={{this.participantExternalId}} />
```

Comment:
Les Input ne sont pas dépréciés ?

Reply:
Nope,  c'est ce qu'on lisait tout à l'heure. seule les arguments le sont.

Si on fait un composant custom extends Checkbox / Input etc... là il faut installer le built in component. (Je n'en ai pas vu donc pour le moment on peut les laisser le temps de passer sur des PixInput.